gui/default/index.html

```diff
@@ -344,40 +344,99 @@ <h3 class="panel-title">
+                <div ng-if="locationDoesNotMatchWebauthnRpId()">
+                  <p class="text-info" translate>Current domain does not match WebAuthn configuration.</p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>Syncthing Documentation: Customizing the GUI address</a>
+                  </p>
+                </div>
+                <p class="text-info" ng-if="isLocationInsecure()" translate>WebAuthn requires HTTPS on non-localhost domains.</p>
+                <p class="text-danger" ng-if="webauthn.errors.notRegistered" translate>This device or security key is not registered. Please try a different one.</p>
+                <p class="text-info" ng-if="webauthn.errors.aborted" translate>Authentication aborted by user.</p>
+                <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>Authentication failed, please try again.</p>
+                <div ng-if="webauthn.errors.securityError">
+                  <p class="text-danger" translate>Initialization blocked by browser. Please make sure Syncthing is correctly configured.</p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>Syncthing Documentation: Customizing the GUI address</a>
+                  </p>
+                </div>
+                <p class="text-danger" ng-if="webauthn.errors.initFailed" translate>Initialization failed, see Syncthing logs for details.</p>
+                <p class="text-danger" ng-if="webauthn.errors.authenticationFailed" translate>Authentication failed, see Syncthing logs for details.</p>
+                <p class="text-info" ng-if="webauthn.errors.noCredentials" translate>No WebAuthn credentials configured.</p>
+                <p class="text-danger" ng-if="webauthn.errors.uvRequired" translate>User verification is required, but was not performed.</p>
+              </div>
+              <div class="panel-footer clearfix">
+                <label class="stay-logged-in">
+                  <input type="checkbox" ng-model="login.stayLoggedIn" ng-disabled="login.inProgress">&nbsp;<span translate>Stay logged in</span>
+                </label>
+
+                <button ng-if="isLocationInsecure() || locationDoesNotMatchWebauthnRpId()" type="button" class="btn btn-primary" ng-click="reloadLoginAtWebauthnAddress()" translate translate-value-webauthn-address="{{ inferWebauthnAddress() }}">Reload page at {%webauthnAddress%}</button>
+
+                <button type="button" class="btn btn-primary pull-right" ng-click="authenticateWebauthnFinish()" ng-disabled="login.inProgress || webauthn.errors.noCredentials || webauthn.errors.initFailed || locationDoesNotMatchWebauthnRpId() || isLocationInsecure()" tabindex="3">
+                  <span translate>Log in</span>
```

Comment:
```suggestion
                  <span translate>Log In</span>
```

Reply:
Fixed in b2f8d69d7512cf382acc0836c171fc5958ca3487.